.changeset/young-geese-teach.md

```diff
@@ -0,0 +1,6 @@
+---
+"barnard59-base": patch
+"barnard59-cube": patch
```

Comment:
I would not include `-cube` in this changeset. It's going to receive a bump anyway, because it depends on `-base`

Reply:
Actually, I was wrong. Given the dependencies, `cube` will not be bumped because the new version `2.3` satisfies the range `^2.2.0`

We have 3 options (from bad to less bad):

1. Change the range to `~2.2.0`
2. Keep what you had before
3. Use experimental changesets option `updateInternalDependents: always`

The first is bad because we don't really want more strict range here. Sure path to duplicate resolutions of what should be compatible versions
Second is ok, but one needs to consider all dependencies every time. 
Last is an unstable feature but will ensure that any release of `b59-base`, for example, will also bump all packages which use it. This may be useful to ensure that we always get latest versions